bucket/bind.json

```
@@ -21,7 +21,35 @@
             "Remove-Item \"$dir\\*\" -Exclude 'bin'"
         ]
     },
-    "env_add_path": "bin",
+    "bin": [
+        "bin/arpaname.exe",
+        "bin/ddns-confgen.exe",
+        "bin/delv.exe",
+        "bin/dig.exe",
+        "bin/dnssec-cds.exe",
+        "bin/dnssec-dsfromkey.exe",
+        "bin/dnssec-importkey.exe",
+        "bin/dnssec-keyfromlabel.exe",
+        "bin/dnssec-keygen.exe",
+        "bin/dnssec-revoke.exe",
+        "bin/dnssec-settime.exe",
+        "bin/dnssec-signzone.exe",
+        "bin/dnssec-verify.exe",
+        "bin/host.exe",
+        "bin/mdig.exe",
+        "bin/named-checkconf.exe",
+        "bin/named-checkzone.exe",
+        "bin/named-compilezone.exe",
+        "bin/named-journalprint.exe",
+        "bin/named-rrchecker.exe",
+        "bin/named.exe",
+        "bin/nsec3hash.exe",
+        "bin/nslookup.exe",
```

To prevent conflict with `C:\WINDOWS\System32\nslookup.exe`

```suggestion
        [
            "bin/nslookup.exe",
            "nslookup-bind"
        ],
```